src/families/tezos/bridge/js.ts

```diff
@@ -159,6 +147,19 @@ const getTransactionStatus = async (
     errors.amount = new NotEnoughBalance();
   }
 
+  // Catch a specific case that requires a minimum amount
+  if (
+    !errors.amount &&
+    t.mode === "send" &&
+    t.amount.lt(EXISTENTIAL_DEPOSIT) &&
+    (await api.getAccountByAddress(t.recipient)).type === "empty"
+  ) {
```

Comment:
I had to move it here because it's actually not inside a "tequito error" (weirdly) but one more edge case to add on top of taquito